.husky/pre-commit-phpcbf.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Disable exit on error since we handle exit codes manually
+set +e
+
+# Get list of staged PHP files
+STAGED_PHP_FILES=$(git diff --cached --name-only --diff-filter=d | grep ".php$") || true
+
+# Exit if no PHP files are staged
+if [ -z "$STAGED_PHP_FILES" ]; then
+    exit 0
+fi
+
+echo "Running PHPCBF on staged files..."
+
+# First try to fix what we can with PHPCBF
+echo "$STAGED_PHP_FILES" | xargs vendor/bin/phpcbf
+
+echo 'does this run?'
+
+# Check the return code
+PHPCBF_STATUS=$?
+
+# If files were fixed, add them back to staging
+if [ $PHPCBF_STATUS -ne 3 ]; then
+    echo "$STAGED_PHP_FILES" | xargs git add
+    
+    if [ $PHPCBF_STATUS -eq 1 ] || [ $PHPCBF_STATUS -eq 2 ]; then
+        echo "\nShowing remaining errors after auto-fixing:"
```

Comment:
The `\n` here probably won't be expanded.
```suggestion
        echo ""
        echo "Showing remaining errors after auto-fixing:"
```